src/plugins/intel_cpu/src/async_infer_request.cpp

```diff
@@ -9,9 +9,24 @@
 ov::intel_cpu::AsyncInferRequest::AsyncInferRequest(
     const std::shared_ptr<IInferRequest>& request,
     const std::shared_ptr<ov::threading::ITaskExecutor>& task_executor,
-    const std::shared_ptr<ov::threading::ITaskExecutor>& callback_executor)
-    : ov::IAsyncInferRequest(request, task_executor, callback_executor) {
+    const std::shared_ptr<ov::threading::ITaskExecutor>& callback_executor,
+    const bool is_single_thread)
```

Comment:
This variable name doesn't completely emphasize the meaning . There should be not only a single thread, but also a single stream, and the second fact is more important, as exactly this fact gives us the possibility to skip overheads connected with the exception handling. Suggest renaming to `is_optmized_single_stream`.

Reply:
updated